testsuite/src/it/java/io/quarkus/ts/startstop/QuarkusMavenPluginTest.java

```diff
@@ -96,6 +96,7 @@ public void infoTarget(TestInfo testInfo) throws IOException, InterruptedExcepti
 
     @Test
     @DisabledOnOs({OS.WINDOWS}) // https://github.com/quarkusio/quarkus/issues/33403
+    @Disabled("https://github.com/quarkus-qe/quarkus-startstop/issues/356")
```

Comment:
There is no description in the issue

And there is https://github.com/quarkus-qe/quarkus-startstop/pull/353 in progress

I would drop this disable line